tron/commands/client.py

```diff
@@ -91,16 +97,27 @@ def build_get_url(url, data=None):
         return url
 
 
+def ensure_user_attribution(headers: Mapping[str, str]) -> Dict[str, str]:
+    headers = headers.copy()
+    if "User-Agent" not in headers:
+        headers["User-Agent"] = USER_AGENT
+    headers["User-Agent"] += f' ({os.environ.get("USER", "anonymous")})'
```

Comment:
i wonder if there's a way for us to get the hostname of the box that a user executed the tronctl-CLUSTER wrapper from in case we can't reach someone that's abusing tron by scripting tronview/tronctl or something :p

Reply:
What if we add a flag to tronctl to add more info to useragent arbitrarily, and then ensure our wrappers can set that ?

Reply:
that's not a bad idea - i had a possibly worse idea that we just update the cluster wrapper to set an ORIG_HOST env var before executing tronctl on the master (since it doesn't seem like SSH exposes anything that we can use for this purpose) and using that value, but your approach seems more extensible if we ever decide we wanna add more info :p